src/BusinessLogic/Bootstrap/Aspect/Aspects.php

```diff
@@ -11,15 +11,20 @@
  *
  * @package SeQura\Core\BusinessLogic\Bootstrap\Aspect
  */
+/**
+ * @phpstan-consistent-constructor
+ */
 class Aspects
 {
     /**
      * @var T|null
      */
+    // @phpstan-ignore-next-line
```

Comment:
Same for this kind of ignores.
Please, could you give some context on the PR message about why we are ignoring this kind of lines and why it is safe to ignore it